lib/hz2600/Kazoo/Api/Entity/Search.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Kazoo\Api\Entity;
```

Comment:
I believe we should implement this as a collection

Reply:
I believe too but I have not tested it as collection so it should be tested before to request pull.

Reply:
Ok, I will close this out for now then. Just open a new pull when you have a version with search implemented as a collection and I will merge it. 

Thank you!